docs/java/features/resilience/resilience.md

```diff
@@ -0,0 +1,221 @@
+---
+id: resilience
+title: Resilience Capabilities of the SAP Cloud SDK for Java
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Resilience
+description: This article describes how the SAP Cloud SDK for Java can be used to introduce resilience patterns into an application.
+keywords:
+- sap
+- cloud
+- sdk
+- resilience
+- multi tenancy
+- cloud native
+- timeout
+image:
+---
+
+The SAP Cloud SDK for Java provides abstractions for some frequently used resilience patterns like timeout, retry or circuit breaker.
```

Comment:
```suggestion
The SAP Cloud SDK for Java provides abstractions for some frequently used resilience patterns like
- timeout
- retry
- circuit breaker.
```

Reply:
I don't want to enumerate here because it might give the false impression that this is a comprehensive list. I really only mention examples here to get the reader into the topic. Having an enumeration in the first sentence would be a bit too disruptive I think.

Reply:
All it does is captures attention and make it easier to read because of more structured representation. Same as with code, but writing is not coding... I'll leave it to your taste. I don't see much false impression as `some frequently used` makes it obvious for me and having it as a list makes it pleasure for my eyes if I want to remind my self what you mentioned again as the most frequently used patterns.